util/regexp-assemble/regexp-942190.data

```diff
@@ -6,7 +6,7 @@
 [\"'`];?\s*?union\b\s*?select\b\s*?[^\s]
 \s*?exec.*?\Wxp_cmdshell
 \s*?execute.*?\Wxp_cmdshell
-\wiif\s*?\(
+\Wiif\s*?\(.+?\)
```

Comment:
While I can see the idea behind matching the full parenthesised string I don't think we should lightly change that part of the regular expression. As you can see in the expressions below, they all explicitly do not match the closing parenthesis. I presume that this was done intentionally. I propose to either only changes the casing of `\w` or to use the same expression to match the contents of the parentheses as the other expressions do.

I could be convinced otherwise, if you could provide an explanation for why this change is good and, especially, why that expression is better than `[^\)]*?`. 

@franbuehler It would actually be great if we could put comments into the data files, explaining why an expression looks a certain way. That would help us immensely with decisions like these.

Reply:
I can understand your point of view, the minimum amendment rules. But I highly recommend: **Please pay attention to the harm of false positives to the production environment**. If you are really worried about the under-reporting after the amendment, I will withdraw the amendment that adds `.+?\)`.  ;-)

Reply:
I don't like FP's either :). Do you have an example of an FP that would be triggered by omitting `.+?\)`?

Reply:
You are right, I did not find the relevant FP. Thinking carefully, `.+?\) `really doesn't help. I agree with your idea .

Reply:
I'm ok with removing the closing bracket again. So let's adjust this PR.

Reply:
Hi @franbuehler, the rules have been updated.